### PR TITLE
Implemented basic aiming indicator

### PR DESCRIPTION
**Pull request by nurgan**
Sorry for total hack, as I said in the slack message.
Works for now.

### CODE REVIEW

File: src/CookieThrower.cpp [hunk 1]
```diff
@@ -12,6 +13,59 @@ CookieThrower::CookieThrower() {
...
+    static bool hasLoaded = false;
+
+    if (!hasLoaded) {
+        shape->loadMesh(RESOURCE_DIR + "sphere.obj");
+        shape->resize();
+        shape->init();
+        hasLoaded = true;
+    }
+
+    BunnyRenderComponent* bunnyRenderComp = new BunnyRenderComponent(shape, progPhong, brass);
```
Comment (joncatanio) [inline src/CookieThrower.cpp:31]: `progPhong` won't allow this program to build. Reed changed the Render components to take a string now. See #33

**Review by joncatanio — CHANGES_REQUESTED** [review]
Doesn't build with master.

File: src/CookieThrower.cpp (hunk 1 above)
Comment (rgarmsen2295) [reply src/CookieThrower.cpp:31]: Just needs to be changed from progPhong to "Phong"

**Comment by rgarmsen2295** [comment]
I'm going to go ahead and merge this and then submit a quick fix to fix the build - @nurgan is busy rn I think and I wanna put the final build through its paces before we demo.

If it's not an easy fix I'll revert

**Comment by rgarmsen2295** [comment]
Seems good after addtl. patch, please verify on your machine

**Pull request merged**